From a6c6d906013f6b5a4019e42a3070c9cadb8e3fb3 Mon Sep 17 00:00:00 2001
From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Wed, 15 Apr 2020 21:44:32 +0200
Subject: [PATCH] data: install appdata to /usr/share/metainfo

---
 data/meson.build | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index 2ffd213..86064a2 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -30,7 +30,7 @@ appstream_file = i18n.merge_file(
   output: 'org.gnome.zbrown.KingsCross.appdata.xml',
   po_dir: '../po',
   install: true,
-  install_dir: join_paths(get_option('datadir'), 'appdata')
+  install_dir: join_paths(get_option('datadir'), 'metainfo')
 )
 
 appstream_util = find_program('appstream-util', required: false)
@@ -67,4 +67,3 @@ install_data('org.gnome.zbrown.KingsCross.Generic.svg',
 
 install_data('org.gnome.zbrown.KingsCross.Generic-symbolic.svg',
              install_dir: join_paths([get_option('datadir'),'icons/hicolor/symbolic/apps']))
-
-- 
2.25.1

